src/pages/PullRequestPage/subroute/FilesChangedTab/FilesChanged/hooks/useImpactedFilesTable.js

```diff
@@ -18,56 +19,73 @@ export const orderingParameter = Object.freeze({
   missesCount: 'MISSES_COUNT',
 })
 
-function getFilters({ sortBy }) {
+function getFilters({ sortBy, flags }) {
   return {
     ordering: {
       direction: sortBy?.desc ? orderingDirection.desc : orderingDirection.asc,
       parameter: orderingParameter[sortBy?.id],
     },
     hasUnintendedChanges: false,
+    ...(flags ? { flags } : {}),
   }
 }
 
 function transformImpactedFilesData({ pull }) {
   const compareWithBase = pull?.compareWithBase
-  const impactedFiles = compareWithBase?.impactedFiles?.map((impactedFile) => {
-    const headCoverage = impactedFile?.headCoverage?.percentCovered
-    const missesCount = impactedFile?.missesCount || 0
-    const patchCoverage = impactedFile?.patchCoverage?.percentCovered
-    const baseCoverage = impactedFile?.baseCoverage?.percentCovered
-    const changeCoverage =
-      isNumber(headCoverage) && isNumber(baseCoverage)
-        ? headCoverage - baseCoverage
-        : Number.NaN
-    const hasHeadOrPatchCoverage =
-      isNumber(headCoverage) || isNumber(patchCoverage)
 
-    return {
-      missesCount,
-      headCoverage,
-      patchCoverage,
-      changeCoverage,
-      hasHeadOrPatchCoverage,
-      headName: impactedFile?.headName,
-      fileName: impactedFile?.fileName,
-      isCriticalFile: impactedFile?.isCriticalFile,
-      pullId: pull?.pullId,
+  const mutatedImpactedFiles = compareWithBase?.impactedFiles?.results?.map(
+    (impactedFile) => {
+      const headCoverage = impactedFile?.headCoverage?.percentCovered
+      const missesCount = impactedFile?.missesCount || 0
```

Comment:
We can get rid of the || 0 here as we always return a number as per https://github.com/codecov/codecov-api/blob/main/graphql_api/types/impacted_file/impacted_file.py#L126C22-L126C22, which follows https://github.com/codecov/codecov-api/blob/main/services/comparison.py#L893. Probably logic copy-pasted from when it wasn't working 👌